scripts/compile/compile-stage.ps1

```diff
@@ -59,6 +61,11 @@ $Projects | ForEach-Object {
     }
 }
 
+if (Test-Path $binariesOutputDir) {
+    cp $binariesOutputDir\* $OutputDir\bin -force -recurse
+    Remove-Item $binariesOutputDir -recurse
```

Comment:
This will blow up incremental compilation because it deletes the output files which will trigger a rebuild next time.